ramls/holdings-sync.raml

```diff
@@ -48,3 +48,29 @@ types:
         body:
           text/plain:
             example: "Internal server error"
+/holdings-storage/batch/synchronous-unsafe:
```

Comment:
As I understand it, RAML files are roughly equivalent to interface definitions.

Given that the unsafe variations of the batch operations are defined in separate interfaces, I think they should also be in separate RAML files.

Reply:
Good point, split.